### PR TITLE
Recommended workouts for docs homepage

### PR DESCRIPTION
**Pull request by StephenBarlow**

### CODE REVIEW

**Comment by StephenBarlow** [comment]
@trevorblades Can we add analytics events to links clicked within workout lists?

File: docs/source/index.mdx [hunk 1]
```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
...
+  icon={Weightlift}
+  keyWord="Develop"
+  otherWords="your graph"
+>
+
+* Caching in [Apollo Server](https://www.apollographql.com/docs/apollo-server/performance/caching/) and [Apollo Client](https://www.apollographql.com/docs/react/caching/cache-configuration/)
+* [Using schema directives](https://www.apollographql.com/docs/apollo-server/schema/directives/) (Apollo Server)
+* [Extending Apollo Server with plugins](https://www.apollographql.com/docs/apollo-server/integrations/plugins/)
+* Mocking new features in [Apollo Server](https://www.apollographql.com/docs/apollo-server/testing/mocking/) and [Apollo Client](https://www.apollographql.com/docs/react/development-testing/client-schema-mocking/)
+* [The schema registry](https://www.apollographql.com/docs/graph-manager/schema-registry/)
```
Comment (zionts) [inline docs/source/index.mdx:89]: Every other bullet point uses gerunds. Could we make this "Registering your schema in the [schema registry]" instead?

Comment (zionts) [reply docs/source/index.mdx:89]: Also is there a good reason for this to be the last bullet point? If we want to promote schema registration as a key part of developing your graph, I think it should be higher on the list.

File: docs/source/index.mdx [hunk 2]
```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
...
+  icon={Paragliding}
+  keyWord="Deploy"
+  otherWords="to production"
+>
+
+* Deploying Apollo Server to [Heroku](https://www.apollographql.com/docs/apollo-server/deployment/heroku/), [AWS Lambda](https://www.apollographql.com/docs/apollo-server/deployment/lambda/), [Zeit Now](https://www.apollographql.com/docs/apollo-server/deployment/now/), etc.
+* Authentication in [Apollo Server](https://www.apollographql.com/docs/apollo-server/security/authentication/) and [Apollo Client](https://www.apollographql.com/docs/react/networking/authentication/)
+* Introduction to [Apollo Federation](https://www.apollographql.com/docs/apollo-server/federation/introduction/) and [Managed federation](https://www.apollographql.com/docs/graph-manager/managed-federation/overview/)
+* [Improving performance](https://www.apollographql.com/docs/react/performance/performance/) (Apollo Client)
+* [Preventing breaking changes to your graph](https://www.apollographql.com/docs/graph-manager/schema-validation/)
```
Comment (zionts) [inline docs/source/index.mdx:103]: Similar to above, could we have this appear higher on the bulleted list?

File: docs/source/index.mdx [hunk 3]
```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
...
+  icon={HeadLight}
+  keyWord="Discover"
+  otherWords="Apollo"
+>
+
+* [The Apollo GraphQL platform](./intro/platform/)
+* [Why GraphQL?](./intro/benefits/)
+* [Introduction to Apollo Server](https://www.apollographql.com/docs/apollo-server/)
+* [Introduction to Apollo Client](https://www.apollographql.com/docs/react/)
+* [Overview of Apollo Graph Manager](https://www.apollographql.com/docs/graph-manager/)
```
Comment (zionts) [inline docs/source/index.mdx:61]: Could I suggest a section on "Getting set up with Apollo tooling" which introduces users to the chrome extension, the CLI, and VS Code extension?

File: docs/source/index.mdx [hunk 4]
```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
...
+
+<WorkoutMenuItem
+  icon={Diving}
+  keyWord="Dive in"
+  otherWords="to the basics"
+>
+
+* Get started with [Apollo Server](https://www.apollographql.com/docs/apollo-server/getting-started/) and [Apollo Client](https://www.apollographql.com/docs/react/get-started/)
+* [Resolvers](https://www.apollographql.com/docs/apollo-server/data/resolvers/) and [Data sources](https://www.apollographql.com/docs/apollo-server/data/data-sources/) (Apollo Server)
+* [Queries](https://www.apollographql.com/docs/react/data/queries/) and [Mutations](https://www.apollographql.com/docs/react/data/mutations/) (Apollo Client)
```
Comment (zionts) [inline docs/source/index.mdx:73]: In the child doc here, I think we should also be covering operation naming as a best practice

File: docs/source/index.mdx [hunk 5]
```diff
@@ -32,3 +43,79 @@ It's easy to adopt Apollo incrementally, meaning you can set it up alongside an
...
+<WorkoutMenuItem
+  icon={BadmintonPlayer}
+  keyWord="Debug"
+  otherWords="your solution"
+>
+
+* Testing in [Apollo Server](https://www.apollographql.com/docs/apollo-server/testing/testing/) and [Apollo Client](https://www.apollographql.com/docs/react/development-testing/testing/)
+* Error handling in [Apollo Server](https://www.apollographql.com/docs/apollo-server/data/errors/) and [Apollo Client](https://www.apollographql.com/docs/react/data/error-handling/)
+* [Metrics and logging](https://www.apollographql.com/docs/apollo-server/monitoring/metrics/) (Apollo Server)
+* [Segmenting metrics by client](https://www.apollographql.com/docs/graph-manager/client-awareness/)
```
Comment (zionts) [inline docs/source/index.mdx:116]: This doesn't really seem like a "debug" step but instead part of "production," no?

**Comment by trevorblades** [comment]
> @trevorblades Can we add analytics events to links clicked within workout lists?

This is implemented in 1d2d48a

File: docs/source/index.mdx (hunk 4 above)
Comment (StephenBarlow) [reply docs/source/index.mdx:73]: Issue created

File: docs/source/index.mdx (hunk 3 above)
Comment (StephenBarlow) [reply docs/source/index.mdx:61]: That's currently under "Dive in". I'll move it up a bit.

**Pull request merged**